main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	var tempChartStoreTargetDir string
 	var disableRemediation bool
 	var cleanupDownloadedCharts bool
+	var runDebug bool
```

Comment:
Where does this get used? Couldn't find a reference across the changeset

Reply:
😆 That was from another change that I was moving to a separate PR. I'll remove this